### ISSUE TITLE
I am getting error while adding dependency of FFMpeg

### ISSUE THREAD

**Issue by sunil-singh-chaudhary** [brarcher/video-transcoder#161]
ERROR: Failed to resolve: protect:android-ffmpeg:release

dependency error pls help how to resolve it

**Comment by licaon-kter** [comment]
How do you want to build? Gradle? Android Studio?

**Comment by sunil-singh-chaudhary** [comment]
i am trying to build and now getting error with this above dependency i use this dependency  implementation 'nl.bravobit:android-ffmpeg:1.1.7@aar'
instead of this  implementation 'protect:android-ffmpeg:release@aar'
but now getting more errors in android studio

**Comment by brarcher** [comment]
The ffmpeg android dependency is from https://github.com/brarcher/ffmpeg-android. There should be a binary build of that project included in https://github.com/brarcher/ffmpeg-android-java, when is then included in Video Transcoder. Building with gradle should work without issue, as that is how I build releases. Do you have local changes you are trying to include, or are you simply building from the master branch?

**Comment by sunil-singh-chaudhary** [comment]
YES actually i find the solution . for others who hv the same error it mainly due to new build.gradle dependency **nl.bravobit:android-ffmpeg:1.1.7@aar**  insist of you can use ### **nl.bravobit:android-ffmpeg:1.1.1@aar**
